### PR TITLE
Tasks: Add smoke test and correct docs/index.rst.

### PR DESCRIPTION
**Pull request by busunkim96**
In preparation for Tasks v2 GA.

### CODE REVIEW

File: tasks/tests/system/gapic/v2/test_system_tasks_v2.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Copyright 2018 Google LLC
```
Comment (plamut) [inline tasks/tests/system/gapic/v2/test_system_tasks_v2.py]: Shouldn't that be 2019?
```suggestion
# Copyright 2019 Google LLC
```

**Comment by plamut** [comment]
@busunkim96 How can this be verified, what is the expected result?

I am probably not the best person to review this, as I am not familiar with Tasks, and also need to prioritize PubSub, sorry.

File: tasks/tests/system/gapic/v2/test_system_tasks_v2.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+class TestSystemTasks(object):
+    def test_recognize(self):
+        client = tasks_v2.CloudTasksClient()
+
+        # Setup Request
+        parent = client.location_path(os.environ["PROJECT_ID"], "us-central1")
+
+        paged_list_response = client.list_queues(parent)
+        resources = list(paged_list_response)
+        assert len(resources) == 0
```
Comment (tseaver) [inline tasks/tests/system/gapic/v2/test_system_tasks_v2.py]: How sure are we that our CI account will never be used to test `google-cloud-tasks` manually, and therefore leave a queue lying around?

Comment (tseaver) [reply tasks/tests/system/gapic/v2/test_system_tasks_v2.py]: Also, [looks like we need to enable the API for our CI account](https://source.cloud.google.com/results/invocations/ff51496d-29ef-4c01-921a-300c65143b98/targets/cloud-devrel%2Fclient-libraries%2Fgoogle-cloud-python%2Fpresubmit%2Ftasks/log).

Comment (busunkim96) [reply tasks/tests/system/gapic/v2/test_system_tasks_v2.py]: Fair point, I'll rework this so it doesn't depend on a certain # of queues being found. 

Also Pinged Chris to enable the Tasks API.

**Pull request merged**